docs/pages/guides/using-nextjs.mdx

```diff
@@ -6,123 +6,222 @@ description: A guide for integrating Next.js with Expo for the web.
 
 import { Collapsible } from '~/ui/components/Collapsible';
 
-> Please open any issues related to Next.js with Expo at [expo-cli/issues](https://github.com/expo/expo-cli/issues).
+> **Warning:** Using Next.js is not an official part of Expo's universal app development workflow.
 
-[Next.js][nextjs] is a React framework that provides simple page-based routing as well as server-side rendering. To use Next.js with Expo for web we recommend that you use a library called [`@expo/next-adapter`][next-adapter] to handle the configuration and integration of the tools.
+[Next.js][nextjs] is a React framework that provides simple page-based routing as well as server-side rendering. To use Next.js with the Expo SDK, we recommend that you use a library called [`@expo/next-adapter`][next-adapter] to handle the configuration.
 
-Using Expo with Next.js means you can share all of your existing components and APIs across your mobile and web. Next.js has its own webpack config so **you'll need to start your web projects with the Next.js CLI and not with `npx expo start`.**
+Using Expo with Next.js means you can share some of your existing components and APIs across your mobile and web app. Next.js has its own CLI that you'll need to use when developing for the web platform, so **you'll need to start your web projects with the Next.js CLI and not with `npx expo start`.**
 
 > Next.js can only be used with Expo for web, this doesn't provide Server-Side Rendering (SSR) for native apps.
 
-## TL;DR:
-
-- Init: `npx create-react-native-app -t with-nextjs` (or `npx create-next-app -e with-expo`)
-- Start: `yarn next dev`
-- Open: `http://localhost:3000/`
-
 ## Setup
 
 To get started, create a new project with [the template](https://github.com/expo/examples/tree/master/with-nextjs):
 
 ```sh
-npx create-react-native-app -t with-nextjs
+npx create-expo-app -e with-nextjs
 ```
 
-- **Web**: `yarn next dev` -- start the Next.js project
 - **Native**: `npx expo start` -- start the Expo project
+- **Web**: `npx next dev` -- start the Next.js project
 
-### Add Next.js to Expo projects
-
-> This is for already existing Expo projects.
+## Manual setup
 
-In this approach you would be using SSR for web in your universal project. This is the recommended path because it gives you full access to the features of Expo and Next.js.
+### Dependencies
 
-<Collapsible summary="Instructions">
+Ensure you have `expo`, `next`, `@expo/next-adapter` installed in your project. `yarn add expo next @expo/next-adapter`
```

Comment:
```suggestion
Ensure you have `expo`, `next`, `@expo/next-adapter` installed in your project:

<Terminal cmd={["$ yarn add expo next @expo/next-adapter"]} />
```

Let's use the `<Terminal>` component instead of `sh` language code snippet to highlight the commands.